assets/queries/k8s/container_security_context/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "container_security_context",
+  "queryName": "Container Security Context",
```

Comment:
Query name should identify the problem with the resource, like in this case, if the container or Pod doesn't have a security context defined